pages.es/linux/eu-readelf.md

```diff
@@ -0,0 +1,12 @@
+# eu-readelf
+
+> Muestra información sobre archivos ELF.
+> Más información: <https://manned.org/eu-readelf>.
+
+- Muestra toda la información extraíble contenida en el archivo ELF:
+
+`eu-readelf --all {{ruta/al/archivo}}`
+
+- Muestra el contenido de todos los segmentos/secciones de NOTE, o de un segmento/sección en particular:
+
+`eu-readelf --notes[={{.note.ABI-tag}}] {{ruta/al/fichero}}`
```

Comment:
I will check if the option can be parsed without an equals sign.

Reply:
I think the option argument cannot be read without the equals sign. Without it, it interprets the option argument as another file.

Steps to recreate:

1. Create a temporary file (e.g. `mktemp --tmpdir /tmp`)
2. Run `eu-readelf --notes FOO` and the pathname of the file

You should see the following output:

```
eu-readelf: cannot open input file 'FOO': No such file or directory
eu-readelf: input file is empty
```

`binutils` version: `2.40-14`

Reply:
so the original source is...

Reply:
I do not understand, could you please elaborate?

Reply:
that is the original line: `eu-readelf --notes[={{.note.ABI-tag}}] {{path/to/file}}`